### PR TITLE
Docs: Move Docs to Monorepo

### PR DESCRIPTION
**Pull request by thomasdax98**
Integrate [comet-docs repo](https://github.com/vivid-planet/comet-docs) into this repo

### CODE REVIEW

**Comment by thomasdax98** [comment]
@johnnyomair Moving the docs to the monorepo works. Also with the Story POC (see https://github.com/vivid-planet/comet/pull/1170).

For the deployment @dkarnutsch (or another admin) has to adjust the Netlify configuration

**Comment by johnnyomair** [comment]
How would I start the docs locally? Could you please add a script to the root package.json and README?

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -69,6 +69,10 @@ jobs:
                   # check for duplicate ids of formatted messages
                   pnpm run intl:extract
 
+            - name: Build Docusaurus
+              run: |
+                pnpm --filter ./docs run build
```
Comment (johnnyomair) [inline .github/workflows/lint.yml:74]: Wouldn't it be a good idea to only build (= lint) the docs if there are changes to the docs?

Comment (thomasdax98) [reply .github/workflows/lint.yml:74]: That would be a good idea. I couldn't find a simple way to skip the step. 

I can create a second workflow that is run only if the docs change. But with #1170 the docs have a dependency on the libraries. Meaning the libraries must be built twice in the Lint workflow and the Build Docs workflow

Comment (johnnyomair) [reply .github/workflows/lint.yml]: We're already building the libs multiple times 😅  See release, release-canary workflows. For the build docs workflow, we should only build the necessary packages, namely `@comet/admin` packages

Comment (thomasdax98) [reply .github/workflows/lint.yml]: Okay, I added a build docs workflow: https://github.com/vivid-planet/comet/actions/runs/5529852385/jobs/10088476382?pr=1168

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**